src/DoctrineBatchUtils/BatchProcessing/SimpleBatchIteratorAggregate.php

```diff
@@ -26,48 +27,29 @@ final class SimpleBatchIteratorAggregate implements IteratorAggregate
      */
     private $batchSize;
 
-    /**
-     * @param AbstractQuery $query
-     * @param int           $batchSize
-     *
-     * @return self
-     */
-    public static function fromQuery(AbstractQuery $query, int $batchSize)
+    public static function fromQuery(AbstractQuery $query, int $batchSize): self
     {
         return new self($query->iterate(), $query->getEntityManager(), $batchSize);
     }
 
-    /**
-     * @param object[]               $results
```

Comment:
This type declaration should remain

Reply:
If not `object[]`, then at least `mixed[]` to make the removal of type restrictions explicit